src/chrome/internal/scripts/htmlToScriptPositionTranslator.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import { createLineNumber, createColumnNumber } from '../locations/subtypes';
+import { Position, LocationInScript } from '../locations/location';
+
+export class HtmlToScriptPositionTranslator {
+    public toPositionRelativeToScript(positionOfScriptTagInHtml: Position, positionRelativeToHtml: LocationInScript): LocationInScript {
+        // All the lines need to be adjusted by the relative position of the script in the resource (in an .html if the script starts in line 20, the first line is 20 rather than 0)
+        const lineRelativeToScript = positionRelativeToHtml.position.lineNumber - positionOfScriptTagInHtml.lineNumber;
+
+        // The columns on the first line need to be adjusted. Columns on all other lines don't need any adjustment.
```

Comment:
Just curious, why is it that only the first line's columns need to be adjusted?

Reply:
script.js:

> console.log('a');
> console.log('b');
> console.log('c');
> 

when turned into an inline script:

```
<script>console.log('a'); <--- all the columns in this line are shifted 8 column numbers to the right because they appear after the script tag
console.log('b'); <--- This one is not shifted
console.log('c');</script> <--- This one is not shifted either
```